dropbox-sdk-dotnet/Dropbox.Api/DropboxRequestHandler.cs

```diff
@@ -816,9 +860,13 @@ internal sealed class DropboxRequestHandlerOptions
         /// </summary>
         private const string BaseUserAgent = "OfficialDropboxDotNetSDKv2";
 
-        public DropboxRequestHandlerOptions(DropboxClientConfig config, string oauth2AccessToken)
+        public DropboxRequestHandlerOptions(DropboxClientConfig config, string oauth2AccessToken, string oauth2RefreshToken, Nullable<DateTime> oauth2AccessTokenExpiresAt, string appKey, string appSecret)
```

Comment:
Would it help if we also keep the old constructor API for creating the `DropboxRequestHandlerOptions` without the SLT related parameters?